icd/generated/mock_icd.cpp

```diff
@@ -193,20 +198,28 @@ static VKAPI_ATTR void VKAPI_CALL DestroyInstance(
     const VkAllocationCallbacks*                pAllocator)
 {
 
-    DestroyDispObjHandle((void*)instance);
+    if (instance) {
+        for (const auto physical_device : physical_device_map.at(instance))
+            DestroyDispObjHandle((void*)physical_device);
+        DestroyDispObjHandle((void*)instance);
```

Comment:
Do you want to erase the `physical_device_map` entry once you are done with it?

Reply:
dammit